imagetagger/imagetagger/images/views.py

```diff
@@ -86,12 +87,36 @@ def index(request):
     ).select_related('team').filter(team__in=userteams).order_by('-priority', '-time')
     imageset_creation_form = ImageSetCreationFormWT()  # the user provides the team manually
     imageset_creation_form.fields['team'].queryset = userteams
+    all_images = Image.objects.all().select_related('image_set').distinct()
```

Comment:
Why would you use distinct here? That is probably an artifact from a previous version using multiple joins and aggregations; this simple query should not contain duplicate rows without it.